samples/bot-types-of-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,136 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send types of cards using bot.
```

Comment:
Demonstrating the feature where user can send different types of cards using bot.

Reply:
fixed